addons/mail/static/src/chatter/web/chatter.xml

```diff
@@ -28,6 +28,9 @@
                     <button class="btn btn-link text-action" t-att-class="{ 'o-active': state.isSearchOpen }" aria-label="Search Messages" title="Search Messages" t-on-click="onClickSearch">
                         <i class="oi oi-search" role="img"/>
                     </button>
+                    <button class="btn btn-link text-action" t-on-click="popoutAttachment" t-if="attachments.length">
+                        <i class="fa fa-window-restore" aria-hidden="Popout" title="Popout"></i>
+                    </button>
```

Comment:
- `t-if` is the most important attribute so we prefer it being the 1st one, so it's clearer to spot at a glance.
- `Popout` => `Pop out Attachments`
- Nitpicking: could use self-closing tag `<i />` rather than `<i></i>`